packages/data-stores/src/plans/test/resolvers.ts

```diff
@@ -66,56 +51,43 @@ describe( 'getSupportedPlans', () => {
 
 		// setPlans call
 		const planDetailsApiData = {
-			body: MOCK_PLAN_DETAILS_API,
+			body: MockData.API_PLAN_DETAILS,
 		};
 		expect( ( iter.next as PlanDetailsApiDataIterator )( planDetailsApiData ).value ).toEqual( {
 			locale: 'en',
 			type: 'SET_PLANS',
-			plans: [ MOCK_PLAN_FREE, MOCK_PLAN_PREMIUM ],
+			plans: [ MockData.STORE_PLAN_FREE, MockData.STORE_PLAN_PREMIUM ],
 		} );
 
 		// setPlanProducts call
 		expect( iter.next().value ).toEqual( {
 			type: 'SET_PLAN_PRODUCTS',
 			products: [
-				MOCK_PLAN_PRODUCT_FREE,
-				MOCK_PLAN_PRODUCT_PREMIUM_ANNUALLY,
-				MOCK_PLAN_PRODUCT_PREMIUM_MONTHLY,
+				MockData.STORE_PRODUCT_FREE,
+				MockData.STORE_PRODUCT_PREMIUM_ANNUALLY,
+				MockData.STORE_PRODUCT_PREMIUM_MONTHLY,
 			],
 		} );
 
 		expect( iter.next().value ).toEqual( {
 			locale: 'en',
```

Comment:
Why don't we use a mocked value here?

Reply:
Good question — I refactored the code to use a mocked value in https://github.com/Automattic/wp-calypso/pull/50492/commits/565f71dbe20297cf9be7a08a2bb6ac22167eca37